streamer/src/sendmmsg.rs

```diff
@@ -24,6 +26,15 @@ pub enum SendPktsError {
     IoError(io::Error, usize),
 }
 
+pub fn batch_send_single_target(
```

Comment:
This doesn't seem like the right home for this function.  Either on or associated with the connection cache would make more sense IMO

Reply:
Have moved the function into the connection_cache module